src/main/java/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -162,9 +163,13 @@ QueryBuilder makeQuery(String prefix, String appId, AccountId accountId, Account
             builder.dataGroups(search.getAllOfGroups(), "IN");
             builder.dataGroups(search.getNoneOfGroups(), "NOT IN");
         }
-        Set<String> callerStudies = context.getCallerStudies();
-        if (!callerStudies.isEmpty()) {
-            builder.append("AND enrollment.studyId IN (:studies)", "studies", callerStudies);
+        
+        // If the caller is a non-admin administrator (researcher, worker, other roles we dream up), then
+        // their organization dictates the accounts they see. We are still making an exception for accounts
+        // with no study associations, while we transition to the new permissions system.
+        Set<String> callerStudies = context.getOrgSponsoredStudies();
+        if (context.isAdministrator() && !context.isInRole(ADMIN) && !callerStudies.isEmpty()) {
+            builder.append("AND enrollment.studyId IN (:studies)", "studies", callerStudies);        
         }
```

Comment:
context.isAdministrator() doesn't seem to be needed, I don't see any tests or code path where this is executed for a participant account. Unit and integration tests pass without it. Removed. Added a test in HibernateAccountDao to verify that this part of query is skipped for admins who otherwise are a member of an organization.